python/sqlflow_submitter/pai/cluster_conf.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def get_cluster_config(attrs):
+    default_map = {
+        "train.num_ps":        0,
+        "train.num_workers":   1,
+        "train.worker_cpu":    400,
+        "train.worker_gpu":    0,
+        "train.ps_cpu":        200,
+        "train.ps_gpu":        0,
+        "train.num_evaluator": 0,
+        "train.evaluator_cpu": 200,
+        "train.evaluator_gpu": 0,
+    }
+    for (k, v) in attrs.entries():
```

Comment:
1. Does the Python `dict` has an `entries` method? It seems should be `items`.
2. We cannot `raise` a `str` in Python3, the object raised must be a instance of `Exception`
3. L27-30 should be an atomic operation. I think the following code is atomic and is readable:
```python
if not all(isinstance(v, int) for v in attr.values()):
    raise SQLFlowDiagnostic("xxx")
default_map.update(attr)
```

Reply:
done